spring-context-support/src/main/java/org/springframework/cache/jcache/JCacheCache.java

```diff
@@ -28,7 +28,7 @@
 
 /**
  * {@link org.springframework.cache.Cache} implementation on top of a
- * {@link javax.cache.Cache} instance.
+ * {@link Cache} instance.
```

Comment:
That full package name is actually on purpose as the Spring Cache abstraction has also a concept of `Cache`.

Reply:
@snicoll If so, it should be `{@link Cache javax.cache.Cache}`, not `{@link javax.cache.Cache}` as the latter will be rendered as just `Cache` in Javadoc. Is it okay to change to `{@link Cache javax.cache.Cache}`?

See https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/cache/jcache/JCacheCache.html

Reply:
Oh dear, good catch. Yes please.